src/Components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import '../Themes/Button.css'; 
+export default class Button extends React.Component{
```

Comment:
The name conflicts with HTMLs `button` element let's name this something else. 
```suggestion
export default class CustomButton extends React.Component{
```